projects/storefrontlib/src/recipes/storefront.module.ts

```diff
@@ -20,7 +20,7 @@ import { StorefrontFoundationModule } from './storefront-foundation.module';
 @NgModule({
   imports: [
     RouterModule.forRoot([], {
-      scrollPositionRestoration: 'enabled',
+      // scrollPositionRestoration: 'enabled',
```

Comment:
Was it intentional?  @tobi-or-not-tobi

Reply:
Good point! I like to drop it, but we were actually thinking of a proper (documented) solution.

Reply:
I've rolled it back and created #9036